homeassistant/components/amcrest/camera.py

```diff
@@ -547,3 +600,22 @@ def _start_tour(self, start):
             log_update_error(
                 _LOGGER, "start" if start else "stop", self.name, "camera tour", error
             )
+
+    def _ptz_control(self, movement, travl_time):
+        """Move or zoom camera in specified direction."""
+        code = _ACTION[_MOV.index(movement)]
+        try:
+            channel = 1
+            hspeed = 0
+            vspeed = "vertical_speed=1"
```

Comment:
This will result in `&arg2=vertical_speed=1`. I don't think it's supposed to be that. I think it should just be `&arg2=1`.

Reply:
My original code was using python-amcrest's functions of "_move_up(action=None, channel=0, vertical_speed=1_) , etc.   The resulting code was an ugly bunch of nested if's, as I wasn't a good enough Python programmer (after 2 days learning!) to map a _movement_ name to the function name dynamically.   

Then I stumbled on the generic _ptz_control_command_ which was great to dynamically specify actions/codes.   It left me a bit confused how to handle the remaining arguments so I tried to pattern them after how the named "move_xxxx" functions were handling them.   Apparently I got this wrong, even though what I had actually worked!  

Thanks for all the corrections and advice!